common/notification/channels/inapp.ts

```diff
@@ -13,6 +16,14 @@ export const inAppChannel: Channel = {
         await ws.sendMessageToUser(to.userID, message);
     },
     canSend: (_notification: Notification, user: User) => {
+        if (!WebSocketService.hasInstance()) {
+            // In the Jobs Dyno (when sending Reminders) we do not have a connection to the Web Dyno which runs the Websocket Server
+            // thus we cannot deliver an In App Notification in that case
+            // This is not great, but hard to fix from an architectural point of view
+            logger.debug(`Did not sent In App Notification as no Websocket Server is available`);
```

Comment:
FYI @timidak ... I totally overlooked this. We run our backend in two different Dynos (aka 'NodeJS processes'), Notifications are sent from both, but only one has the websocket server running. Thus In App Notifications will not work for reminders for the moment. I have no idea currently how to fix that, and for the time being I guess we can keep it this way until we have reminders were In App Notifications are actually relevant.

Reply:
Thanks for the heads-up @Jonasdoubleyou! Let's have a quick chat about this, maybe on wednesday?

Reply:
Hm, I'm busy the rest of the week. So either asynchronous via Chat or next week monday?